tests/tasks/test_employee_salary_history.py

```diff
@@ -108,9 +108,105 @@ def test_task_transform_employee_salary_history_set_base_change_percent(
         [
             0.0,  # first position, so no change
             0.03,
-            0.456,
+            0.45631,  # 5 digits of accuracy, but rounded to 2 digits in Quickbase
             0.0,  # new position, so no change
             0.125,
             0.1,
         ],
     )
+
+
+def test_task_transform_employee_salary_history_set_effective_change_percent_no_temps(
```

Comment:
Hmm, for completeness, can we also include an assertion for the `effective_salary` column?

Reply:
I see you're thinking here @jonavellecuerdo, but the values for the `effective_salary` are actually tested as part of this test here: https://github.com/MITLibraries/hrqb-client/pull/177/files#diff-260cf8b13f4efa8a193455f2e744008f6124140071150436a72720eb9f0c4e9bR192-R201

The `effective_salary` values -- while needed to calculate the change percentage -- are not actually written to Quickbase (it's a "formula" field there).  

Are you comfortable with how this test _does_ check the intermediate `effective_salary` values?  It's consistent with the [pre-existing test](https://github.com/MITLibraries/hrqb-client/pull/177/files#diff-260cf8b13f4efa8a193455f2e744008f6124140071150436a72720eb9f0c4e9bR143-R164) that checks first the value being considered, and then the calculated percentage change.

Reply:
I think I follow-- if the assertions on the `previous_effective_salary` and `effective_change_percent` pass, then it's safe to assume `effective_salary` are the values you expected. I'm prolly just not quick with back-calculating percent changes 😅 , but I think I'm okay without the changes~

```
np.testing.assert_array_equal(
        new_df.effective_salary.values,
        [
            10_000,
            10_300,
            15_000,
            20_000,
            22_500,
            24_750
        ],
    )
```

Reply:
Merged, but just to close the loop and respond.

Ahhhh.... just deleted my response. I see your point!  You were talking about asserting that the `effective_salary` values are what we expect, specifically testing this logic:

```python
new_salary_df["effective_salary"] = (
    new_salary_df["temp_change_base_amount"]
    .replace(0, np.nan)
    .fillna(new_salary_df["original_base_amount"])
)
```

That's straight-forward enough, I don't think it needs testing.  If it's 0 (zero), then we replace with `np.nan`, and then followup that by filling `np.nan` with the `original_base_amount`. 

We kind of _indirectly_ check them in the code I shared above, by checking the "previous" form of each, which is just one row behind.  

I take your point though: I'll make a note to revisit these tests.  Want to get this deployed for review, but they would benefit from another pass.